client/ayon_kitsu/hooks/pre_status_cahnge.py

```diff
@@ -0,0 +1,31 @@
+from ayon_applications import PreLaunchHook
+import os
+
+class PreStatusChange(PreLaunchHook):
+    order = 1
+    launch_types = ()
```

Comment:
You should keep this only for `local`. This would change status even if application is prepared for farm.